### ISSUE TITLE
Remove PyPI rights for no longer active contributors?

### ISSUE THREAD

**Issue by icemac** [zopefoundation/meta#233]
I came across https://sethmlarson.dev/removing-maintainers-from-open-source-projects.
For many `zopefoundation` projects we have a long list of former contributors who are sill allowed to publish new releases to PyPI.

I think it would be a good idea to ask at least some of them, whether they still need these rights and then remove them.

What do you think?

**Comment by dataflake** [comment]
I agree. If PyPI wouldn't be so "noisy" with emails when someone is removed I would have started that without asking the people.
